releasenotes/notes/gradient-based-adapt-vqe-aa4e89f44c5aa69e.yaml

```diff
@@ -0,0 +1,11 @@
+---
+features:
```

Comment:
Perhaps we should note the default behavior change in the release note so as to alert anyone that used a default delta value that the behavior is different. Maybe it suffices to just highlight this that here and link/refer to the note in the class for more info.